code/routers/country_page/country.js

```diff
@@ -0,0 +1,107 @@
+const express = require('express')
+const router = express.Router();
+
+const Country = require('../../models/country');
+
+const countryCodes = require('../../config/resCodes').country;
+
+
+/*
+  Insert country info api
+
+  inputs:
+    name: String
+    abbreviation: String
+    generalInformation: Collection
+    statistics: Collection
+
+  Inserts the country information into the country database schema
+  This is just a temporary api to insert data into the country schema
+*/
+
+
+router.post('/insert_country_info', (req, res) => {
```

Comment:
For the api call formatting can you change it to kabob case: insert_country_info -> insert-country-info

Reply:
Done.